IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -308,7 +309,7 @@ - (MSIDAccount *)accountForIdentifier:(MSIDAccountIdentifier *)accountIdentifier
                               context:(id<MSIDRequestContext>)context
                                 error:(NSError **)error
 {
-    MSID_LOG_VERBOSE(context, @"(Default accessor) Looking for account with client ID %@, family ID %@, authority %@", configuration.clientId, familyId, configuration.authority);
+    MSID_LOG_VERBOSE(context, @"(Default accessor) Looking for account with client ID %@, family ID %@, authority %@, hashed legacyAccountid %@, hashed homeAccountId %@", configuration.clientId, familyId, configuration.authority, [accountIdentifier.legacyAccountId msidTokenHash], [accountIdentifier.homeAccountId msidTokenHash]);
```

Comment:
Can you store pre-calculated hashes in the MSIDAccountIdentifier object? I believe we do a similar thing in latest common core. Otherwise, we'd do a lot of hash calculation every time when something is logged.

Reply:
addressed. Just to clarify: I looked up at the code in common-core using by MSAL and I see the hashed property there. But I believe every time we call that property, the hash is calculated so no cache. I added a temp cache in this version of code, please have a look (it will be wrong if the value of id changed, but I don't think we are really re-assigning that property.)